doc/sphinx/source/projects/ad7124_iio/ad7124_iio.rst

```diff
@@ -71,6 +71,10 @@ Connect the VIO_ADJUST jumper on the SDP-K1 / JP4 on Nucleo-H563ZI board to 3.3V
 * Connect a male-to-male jumper wire between D8 and D12 on Arduino connectors. (Required for data capture)
 * If using analog inputs AIN0/AIN1, remove the noise test LK1 jumper.
 
+.. note::
+   Set capture_timeout of iio oscilloscope atleast 25s to avoid timing out of the application when capturing all channels at lower odr. 
```

Comment:
Instead of lower odr, should we mention typical/specific numbers here, perhaps?

Reply:
It is really application specific here, there are a few projects that might work with lower odr because they spend less time in the trigger handler.  How do we go about this then?

Reply:
Makes sense to keep the data_capture_and_dmm.rst generic enough then. 
But ad7124_iio.rst is a project-specific file though, so I was thinking something like - 
> at lower ODRs, typically below X samples per second. 

Or if you think that adding that info wouldn't really mean much to a user, as 25seconds timeout would take care of all possible configurations and ODR (for AD7124), that sounds fine to me too. We can keep the phrasing the way it is.

Reply:
For example in my project, for odr of 4800 4 channels can be captured without IIO osc timing out, for odr of 2400 2 channels can be captured, for odr of 19200, 12 channels can be captured if I am not wrong. I think it is better to have the user set 25s and not have to worry about all this

Reply:
Fair enough, sounds good to me